diff --git a/helpers/authproto_pam.c b/helpers/authproto_pam.c
index d625278..c1a653a 100644
--- a/main.c
+++ b/main.c
@@ -1057,7 +1057,7 @@ int main(int argc, char **argv) {
   // Need to flush the display so savers sure can access the window.
   XFlush(display);
 
-  enum WatchChildrenState xss_requested_saver_state = WATCH_CHILDREN_NORMAL;
+  enum WatchChildrenState xss_requested_saver_state = WATCH_CHILDREN_FORCE_AUTH;
   int x11_fd = ConnectionNumber(display);
 
   if (x11_fd == xss_sleep_lock_fd && xss_sleep_lock_fd != -1) {
