diff -ur debian.orig/rules debian/rules
--- debian.orig/rules	2014-02-19 15:13:03.487704331 -0500
+++ debian/rules	2014-02-19 15:13:05.935716470 -0500
@@ -5,8 +5,8 @@
 
 override_dh_auto_build:
 	dh_auto_build
-	cd scintilla/gtk; $(MAKE) GTK3=1 $(BUILD_LUA)
-	cd scite/gtk; $(MAKE) gnomeprefix=/usr GTK3=1 $(BUILD_LUA)
+	cd scintilla/gtk; $(MAKE) $(BUILD_LUA)
+	cd scite/gtk; $(MAKE) gnomeprefix=/usr $(BUILD_LUA)
 
 override_dh_auto_clean:
 	dh_auto_clean
@@ -19,7 +19,7 @@
 
 override_dh_auto_install:
 	dh_auto_install
-	cd scite/gtk; $(MAKE) install GTK3=1 gnomeprefix=../../debian/scite/usr datadir=../../debian/scite/usr/share
+	cd scite/gtk; $(MAKE) install gnomeprefix=../../debian/scite/usr datadir=../../debian/scite/usr/share
 	#chmod -x $(CURDIR)/debian/scite/usr/share/scite/*
 	# needed by the debian-menu file
 	install -m 644 $(CURDIR)/debian/Sci32M.xpm $(CURDIR)/debian/scite/usr/share/scite
